packages/components/src/components/select/select/select.tsx

```diff
@@ -174,7 +165,7 @@ export class Select {
     document.addEventListener('mousedown', this.onClickOutside, true);
     this.form = getClosestHTMLElement(this.host, 'form');
     this.isWithinForm = !!this.form;
-    this.isNativePopoverCase = detectNativePopoverCase(this.host, false);
+    this.isNativePopoverCase = !!detectNativePopoverCase(this.host, false);
```

Comment:
boolean casting shouln't be necessary as `detectNativePopoverCase` returns a boolean